dev-libs/libdivecomputer/libdivecomputer-0.6.0.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+if [[ ${PV} = *9999* ]]; then
+	EGIT_REPO_URI="https://github.com/libdivecomputer/libdivecomputer"
+	inherit autotools git-r3
+	KEYWORDS=""
```

Comment:
Please dont specify KEYWORDS at all here, it messes up with `ekeyword` tool when it's defined twice in an ebuild.